### PR TITLE
Bug squashing V2

### LINKED ISSUE #193: Traitor crossbow doesn't show up in attack logs
The energy crossbow. Maybe @stuicey forgot to log it. Don't know if it shows up in the STUI. Gutsy told me the only place the attacks show up is in the mob variables of the victim.

### LINKED ISSUE #418: Monkeys and backpacks
Monkeys can be picked up, and have backpacks equipped to them (containing any item which a backpack could normally contain), but (of course) backpacks cannot be placed in eachother. However, monkeys **wearing backpacks** can be placed in backpacks.
This means that you can increase your storage space by as many backpacks as monkeys you can obtain.
I'm pretty damn sure breaking reality should be fixed.

### LINKED ISSUE #446: Prevent shuttle vote when shuttle already has been called.
Title.

### LINKED ISSUE #459: Engine blastdoor see-through
Blastdoor is by default shut on round start. Open it and then close it again and it will leave a see-through gap:
gap

### LINKED ISSUE #490: Artifact Creation
As a regular miner with a normal pickaxe, you can mine and walk away from the strange rocks and keep doing so until it gives you an artifact. It seems to be that the rock reloads each time you hit it, and it can give you infinite artifacts.

### LINKED ISSUE #527: Small intercom issue.
In security's locker room the intercom next to the entrance to the vault has its microphone on and speaker off at round start unlike all other intercoms where speaker is on and microphone is off. Although it is funny when you trick someone into talking in the locker room and everyone can hear it except for him.

### LINKED ISSUE #530: Hide verb is broken
Hide verb is broken. Have tried it as mice, alien larvae, and diona nymphs. Works for none of them.

### LINKED ISSUE #532: Users are able to make votes when the round hasn't started
Title.

Already fixing, just making this for eh.. "paperwork" reasons.

### LINKED ISSUE #535: Electricity doesn't hurt people
Shocked doors, hacked machines, etc., do no damage to a player without insulated gloves. It has been this way for some time. The only way someone is shocked is if they accidentally click a powered wire or electrified grille.

### PR DESCRIPTION
**Pull request by runedog48**
- Fixes #459 ( Engine blast door has proper opacity now )
- Fixes #193 ( Powered crossbow generates logs )
- Fixes #527 ( Equipment Storage station radio settings )
- Fixes #490 ( Can't move when excavating with a pickaxe )
- Fixes #446 ( Crew transfer vote makes proper checks before allowing vote )
- Fixes #532 ( Disables pre-game votes )
  - Tweaks 0c8eeca
- Fixes #530 ( Hide ability fixed, now works again )
- Fixes #418 ( Monkeys can no longer be stored in backpacks )
- Fixes #535 ( Getting shocked works again )

Merge at will

### CODE REVIEW

**Comment by BlueSushii** [comment]
Wow rune dog your destroying all the support tickets.

**Comment by halworsen** [comment]
poor issues think of their families :cry:

**Comment by lmas** [comment]
I'm loving your work, good job. But could you prefix your commits with "Fix #issue_number: " so the issues will be closed automagically when we merge?

**Comment by runedog48** [comment]
Ah, I wasn't aware that was a thing. I'll try to do that in the future. Is there a way to edit the pre-existing commits so they follow that format?

**Comment by lmas** [comment]
It's a [special feature of github](https://help.github.com/articles/closing-issues-via-commit-messages/). If you only add a "#issue_number" anywhere in the commit msg it will make a reference to the issue without closing it.

Short version: No, you can't edit commits afterwards.
Long version: Yes, with some weird and dangerous git magic you can edit the commit history on your local repo (not on remotes). Run `git commit --amend` to edit the **latest** commit you did. You'll be prompted to type in a commit message again (old message is prefilled already). But it will also attempt to add any new staged changes to the commit too, so you have to be carefull with it so you don't accidently add new changes to the old commit (and apparently you can do some really dangerous `git rebase` magic too, with emphasis on dangerous. But it's some weird shit way above my level).

TL;DR: Git is weird.

**Comment by runedog48** [comment]
I've tried some Git magic before, and ended up erasing about 4 commits, and the only reason I was able to recover them was because they were stored in memory by Dream Maker because I didn't close the files yet, or else I would have been screwed. Let's just say I'm not attempting any Git magic again.

**Comment by lmas** [comment]
Yep, it's never recommended to try to edit the commit history. It also defeats the whole purpose with git in a way, since in theory you can't trust the commit history after having been messing with it.

**Comment by runedog48** [comment]
Edit: NOT READY

**Comment by lmas** [comment]
You're making quite a big PR again (over 700 files! woo :D), could you please clean things up instead of adding in more fixes and we'll merge this in ASAP? Can't keep up with all your fixes (which is great) :D

Edit: Looking at the list of 700 changed files it just looks like it's because this branch for this PR is not up to date with master. Gonna see what happens when I try to merge in this PR on a local repo.
Other then that your fixe looks great and is of high quality. :+1:

**Comment by Kwask** [comment]
He accidentally reverted a lot of changes, hence the massive list of changes. Hopefully it wont be too painful to fix :P

**Comment by lmas** [comment]
All I see is 2 commits that reverted his own map changes. There's no trace of a revert for the rest of the 700 files?

**Comment by Kwask** [comment]
I'm not sure where the commit is, but looking at the files changed list, a lot of things like the alloy lab / SQL whitelists were reverted

**Comment by runedog48** [comment]
Just got home, been gone all day because of varsity basketball. I'll try to fix this shit.

**Comment by runedog48** [comment]
All ready to go.

**Comment by Kwask** [comment]
:+1:
